### PR TITLE
Feat/fixture doc

### PR DESCRIPTION
**Pull request by SaezChristopher**
Add fixtures documentation :).

### CODE REVIEW

File: fluent/fixtures.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+currentMenu: fluent-fixtures
+---
+
+# Fixtures
+Sometimes, you may need to prepopulate your database with existing datas. For exemple, you might need to insert in your table 'Users' an admin account or a test account.
```
Comment (vzsg) [inline fluent/fixtures.md]: Typos: "existing data**s**" and "ex**e**mple".
Style-wise, you should drop the "Sometimes", "may" and "might" words. Let's make it clear and direct.

File: fluent/fixtures.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            let query = Query<T>(self)
+            query.action = .create
+            query.data = try model.makeNode(context: context)
+            try driver.query(query)
+        }
+    }
+}
+```
+
+In your model User.swift file
```
Comment (vzsg) [inline fluent/fixtures.md]: You used a colon `:` before the previous code snippet, so you should add it at the end of the line here as well.

File: fluent/fixtures.md (hunk 1 above)
Comment (SaezChristopher) [inline fluent/fixtures.md]: I corrected the typos and change the sentence to make it clearer. Thx for advice.

File: fluent/fixtures.md (hunk 2 above)
Comment (SaezChristopher) [inline fluent/fixtures.md]: Indeed. Fixed

**Comment by tanner0101** [comment]
Thanks!

**Pull request merged**